src/main/java/com/halo/core_bridge/api/jobposting/model/dto/JobPostingDto.java

```diff
@@ -0,0 +1,104 @@
+package com.halo.core_bridge.api.jobposting.model.dto;
+
+import com.halo.core_bridge.api.jobposting.model.entity.JobPosting;
+import com.halo.core_bridge.api.jobposting.model.entity.JobPostingSkill;
+import com.halo.core_bridge.api.organization.model.entity.Department;
+import lombok.*;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class JobPostingDto {
+
+    @Getter
+    @Setter
+    @NoArgsConstructor
+    @AllArgsConstructor
+    @Builder
+    public static class CreateRequest {
+        private String title;
+        private String description;
+        private String employmentType;
+        private String careerType;
+        private int minExperience;
+        private int maxExperience;
+        private Long departmentId;
+        private LocalDateTime applyStartDate;
+        private LocalDateTime applyEndDate;
+        private LocalDateTime hireEndDate;
+        private List<String> skills;
+
+        public JobPosting toEntity(Department department) {
+            JobPosting posting = JobPosting.builder()
+                    .title(this.title)
+                    .description(this.description)
+                    .employmentType(this.employmentType)
+                    .careerType(this.careerType)
+                    .minExperience(this.minExperience)
+                    .maxExperience(this.maxExperience)
+                    .applyStartDate(this.applyStartDate)
+                    .applyEndDate(this.applyEndDate)
+                    .hireEndDate(this.hireEndDate)
+                    .department(department)
+                    .build();
+
+            if (skills != null) {
+                this.skills.forEach(skillName ->
+                        posting.addSkill(JobPostingSkill.builder().name(skillName).build()));
+            }
+            return posting;
+        }
+
+    }
```

Comment:
`Department`는 이전 리뷰에서 말씀드린 것처럼,
**연관관계 매핑 시 Id 값만으로도 참조가 가능하기 때문에**
**파라미터로 `Department` 객체를 직접 받지 않고**
DTO에 포함된 departmentId를 이용해 **내부에서 엔티티를 생성**해주시면 좋을 것 같아요.

이렇게 하면 불필요한 `findById()` 쿼리를 줄일 수 있고,
`toEntity()` 메서드 자체가 더 독립적으로 동작할 수 있어요.

```suggestion
        public JobPosting toEntity(Department department) {
            JobPosting posting = JobPosting.builder()
                    . . .
                    .department(
                      // Department 빌더 패턴을 사용해서 DepartmentId만 담긴 Department 엔티티를 저장하면 좋을 것 같아요!
                    )
                    .build();

            . . .

    }
```